lib/utils/crypto_util.dart

```diff
@@ -34,9 +34,9 @@ Uint8List cryptoPwHash(Map<String, dynamic> args) {
   );
 }
 
-Uint8List cryptoGenericHash(Map<String, dynamic> args) {
+Future<Uint8List> cryptoGenericHash(Map<String, dynamic> args) async {
```

Comment:
The caller for this should now await?

Reply:
This is the only place where it's being called. I am not sure if we need to change anything else. Fwiw, we are passing async method to computer inside local_sync_util.
```code=dart
static Future<Uint8List> getHash(io.File source) {
    return _computer.compute(
      cryptoGenericHash,
      param: {
        "sourceFilePath": source.path,
      },
    );
  }
```